Corona-Warn-App/src/main/res/navigation/nav_graph.xml

```diff
@@ -815,7 +815,7 @@
             app:argType="string" />
         <argument
             android:name="correctionLevel"
-            android:defaultValue="Q"
+            android:defaultValue="M"
```

Comment:
Why did you change this? The change should be about the generated QR Code, not the scanned one.

Reply:
@thomasaugsten, @mlenkeit  what do you think?

Reply:
@harambasicluka  the qroode should not change when you got to full screen mode